server/features/steps/steps.py

```diff
@@ -88,6 +88,16 @@ def test_key_is_present(key, context, response):
         '"%s" should be empty or false, but it was "%s" in (%s)' % (key, response[key], response)
 
 
+def test_key_is_not_present(key, response):
```

Comment:
There's a similar function on this line...
https://github.com/superdesk/superdesk/blob/master/server/features/steps/steps.py#L887